tests/post_training/test_templates/test_smooth_quant.py

```diff
@@ -268,12 +264,10 @@ def test_get_activation_channel_axis(self, node_metatype, layer_attributes, port
         }
         node = NNCFNode(attributes)
 
-        try:  # noqa
+        try:
             activation_channel_axis = backend.get_activation_channel_axis(node, port_id)
-        except nncf.InternalError as e:  # noqa
-            # TODO: this is wrong, should only expect an exception for cases where the exception is known to be raised
-            #  with pytest.raises(...)
-            pytest.xfail("FIXME")
+        except nncf.InternalError:
+            pytest.xfail("Expected exception")
```

Comment:
test not fail if `get_activation_channel_axis` raise nncf.InternalError for not expected case. 

```python 
if reference_value is nncf.InternalError:
    with pytest.raises(nncf.InternalError, match="can not take more than")
        backend.get_activation_channel_axis(node, port_id)
else:
    activation_channel_axis = backend.get_activation_channel_axis(node, port_id)
    assert activation_channel_axis == reference_value
```